static/gsApp/views/amCheckout/steps/setPayAsYouGo.tsx

```diff
@@ -378,12 +379,12 @@ const StyledPanelBody = styled(PanelBody)`
 const TwoColumnContainer = styled('div')<{
   alignItems?: string;
   columnWidth?: string;
-  gap?: string;
+  gap?: Space;
   justifyContent?: string;
 }>`
   display: grid;
   grid-template-columns: repeat(2, ${p => p.columnWidth || 'auto'});
-  gap: ${p => p.gap || space(4)};
+  gap: ${p => p.theme.space[p.gap ?? '3xl']};
```

Comment:
was: `30px` (space4)
now: `32px` (3xl)

Reply:
Right, this was an intentional change! `2xl => 24px (was 20px)` and `3xl => 32px (was 30px)`.